internal/data/organizations.go

```diff
@@ -116,7 +118,8 @@ func (ou *OrganizationUpdate) areAllFieldsEmpty() bool {
 		ou.SMSRegistrationMessageTemplate == nil &&
 		ou.OTPMessageTemplate == nil &&
 		ou.SMSResendInterval == nil &&
-		ou.PaymentCancellationPeriodDays == nil)
+		ou.PaymentCancellationPeriodDays == nil &&
+		ou.PrivacyPolicyLink == nil)
```

Comment:
I just wanted to let you know that you are not validating this field. It should only accept a URL.